stdlib/messaging/kafka/src/main/java/org/ballerinalang/messaging/kafka/serdes/BallerinaKafkaDeserializer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.messaging.kafka.serdes;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.ballerinalang.jvm.BRuntime;
+import org.ballerinalang.jvm.values.ObjectValue;
+import org.ballerinalang.jvm.values.api.BArray;
+import org.ballerinalang.jvm.values.api.BValueCreator;
+import org.ballerinalang.messaging.kafka.utils.KafkaConstants;
+
+import java.util.Map;
+
+import static org.ballerinalang.messaging.kafka.utils.KafkaConstants.BALLERINA_STRAND;
+
+/**
+ * Represents a deserializer class for ballerina kafka module.
+ */
+public class BallerinaKafkaDeserializer implements Deserializer {
+
+    private ObjectValue deserializerObject = null;
+    private BRuntime runtime = null;
+    private int timeout = 10000;
+
+    @Override
+    public void configure(Map configs, boolean isKey) {
```

Comment:
Normally 'isKey' kind word use for a method name.  Shall we use something else like keyPresent  etc.?

Reply:
This is the convention used in Kafka [Serializers](https://kafka.apache.org/20/javadoc/org/apache/kafka/common/serialization/Serializer.html#configure-java.util.Map-boolean-) / [Deserializers](https://kafka.apache.org/21/javadoc/org/apache/kafka/common/serialization/Deserializer.html#configure-java.util.Map-boolean-). Shall we keep it this way ?